gem/lib/pupperware/spec_helper.rb

```diff
@@ -110,8 +110,10 @@ def docker_compose(command_and_args, stream: StringIO.new)
                                 #{command_and_args}", stream: stream)
   end
 
-  def docker_compose_up()
+  def docker_compose_up(fix_windows_volumes: [])
```

Comment:
Providing a list here seems unnecessary.

Just ask docker-compose for the volumes in this stack and fix them all!